docker_cli/_mount.py

```diff
@@ -0,0 +1,48 @@
+import os
+from typing import Union
+
+
+class Mount:
+    """A description of a docker mount."""
+
+    def __init__(
+        self,
+        host_mount_point: Union[os.PathLike[str], str],
+        image_mount_point: Union[os.PathLike[str], str],
+        permissions: str
```

Comment:
I think it'd be reasonable for this parameter to default to read/write mode since that's the default mode that docker itself uses.

```suggestion
        permissions: str = "rw"
```